mkosi/distributions/opensuse.py

```diff
@@ -32,46 +32,65 @@ def initrd_path(kver: str) -> Path:
         return Path("boot") / f"initrd-{kver}"
 
 
-def zypper_addrepo(state, url, name, caching=False):
-    run(["zypper", "--root", state.root, "addrepo", "-c", "-k" if caching else "-K", url, name])
-
+def run_zypper(state, global_opts, verb, verb_opts, *args, with_apivfs=False):
```

Comment:
We use `invoke_<package-manager>` everywhere else so this should be named similar

Reply:
with_apivfs is not used? Can't we just always use `run_apivfs` to keep things simple?

Reply:
It is used, `git grep with_apivfs` should tell you where it's needed... or maybe I'm missing your point.

Reply:
I meant inside of `run_zypper()`, it's not used inside the function unless github is showing me the wrong diff

Reply:
ouch, yeah you're right ;)